ckan/logic/schema.py

```diff
@@ -411,6 +411,7 @@ def default_user_schema():
         'reset_key': [ignore],
         'activity_streams_email_notifications': [ignore_missing],
         'state': [ignore_missing],
+        'extras': [ignore_missing, dict]
```

Comment:
data arrives at validators from a number of directions. The API controller is one, but it's also passed arbitrary python objects from any ckan code or plugin calling call_action. This is why we have converters that handle dates being passed to some fields.

The code above doesn't assume a string, it just tests that it's something we can serialize as JSON with dumps, and the topmost thing is an object